ResearchKit/ActiveTasks/ORKActiveStepQuantityView.m

```diff
@@ -245,12 +242,12 @@ - (instancetype)initWithFrame:(CGRect)frame {
         [self addSubview:_leftView];
         [self addSubview:_rightView];
         [self addSubview:_metricKeyline];
-        [self setupConstraints];
+        [self setUpConstraints];
```

Comment:
Will this code make a loop?

`setUpConstraints` -> `setNeedsUpdateConstraints` -> `updateConstraints` -> `setUpConstraints`